src/protagonist/Portal/Pages/Shared/_ControlPanelLayout.cshtml

```diff
@@ -1,123 +1,116 @@
-  <script src="~/assets/dist/js/bootstrap.bundle.min.js"></script>
-  <script src="https://cdn.jsdelivr.net/npm/feather-icons@4.28.0/dist/feather.min.js" integrity="sha384-uO3SXW5IuS1ZpFPKugNNWqTZRRglnUJK6UAZ/gxOX80nxEkN9NcGZTftn6RzhGWE" crossorigin="anonymous"></script>
-  <script src="~/js/portal.js" asp-append-version="true"></script>
-  @await RenderSectionAsync("Scripts", required: false)
-  </body>
+﻿@using DLCS.Web.Auth
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="">
+    <title>@ViewData["Title"] - DLCS Portal</title>
+    <link href="~/assets/dist/css/bootstrap.min.css" rel="stylesheet" />
+    <style>
+      .bd-placeholder-img {
+        font-size: 1.125rem;
+        text-anchor: middle;
+        -webkit-user-select: none;
+        -moz-user-select: none;
+        user-select: none;
+      }
+
+      @@media (min-width: 768px) {
+        .bd-placeholder-img-lg {
+          font-size: 3.5rem;
+        }
+      }
+    </style>
+    <link href="~/css/portal.css" rel="stylesheet" />
+    @await RenderSectionAsync("header", false)
+  </head>
+  <body>
+    
+  <header class="navbar navbar-dark sticky-top bg-dark flex-md-nowrap p-0 shadow">
+    <a class="navbar-brand col-md-3 col-lg-2 me-0 px-3" href="/">DLCS Portal</a>
+    <button class="navbar-toggler position-absolute d-md-none collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#sidebarMenu" aria-controls="sidebarMenu" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <ul class="navbar-nav px-3">
+      @await Html.PartialAsync("_LoginPartial")
+    </ul>
+  </header>
+  
+
+<div class="container-fluid">
+  <div class="row">
+    <nav id="sidebarMenu" class="col-md-3 col-lg-2 d-md-block bg-light sidebar collapse">
+      <div class="position-sticky pt-3">
+        <ul class="nav flex-column">
+          <li class="nav-item">
+            <navlink href="/" icon="home">Home</navlink>
+          </li>
+          <!--
+          <li class="nav-item">
+            <navlink href="/queue" icon="list">Queue</navlink>
+          </li>
+          -->
+          <li class="nav-item">
+            <navlink href="/spaces" icon="folder">Spaces</navlink>
+          </li>
+          <!--
+          <li class="nav-item">
+            <navlink href="/storage" icon="hard-drive">Storage</navlink>
+          </li>
+          <li class="nav-item">
+            <navlink href="/origins" icon="globe">Origins</navlink>
+          </li>
+          <li class="nav-item">
+            <navlink href="/auth" icon="lock">Access control</navlink>
+          </li>
+          <li class="nav-item">
+            <navlink href="/queries" icon="search">Named queries</navlink>
+          </li>
+          -->
```

Comment:
commented out code

Reply:
Removed in [fc69e75](https://github.com/dlcs/protagonist/pull/640/commits/fc69e7553fd562d307aa1a6bc4b797d9a89a6b42)